dashboard/app/views/reports/prizes.html.haml

```diff
@@ -26,12 +27,12 @@
       %br/
 
   - if current_user.teacher_prize
-    %h4= t('redeem_prizes.msg.teacher.earned', num_trophies: Trophy::TROPHY_MAX)
+    %h4= t('redeem_prizes.msg.teacher.earned', num_trophies: trophy_max)
```

Comment:
I want to avoid changing this legacy prizes page — I expect this page will soon go away entirely.